proto/regen/ecocredit/v1/tx.proto

```diff
@@ -273,6 +311,128 @@ message MsgCreateProjectResponse {
   string project_id = 1;
 }
 
+// MsgCreateUnregisteredProject is the Msg/CreateUnregisteredProject request type.
+message MsgCreateUnregisteredProject {
+  option (cosmos.msg.v1.signer) = "creator";
+
+  // creator is the address of the account creating the project that will become
+  // the admin of the project upon creation.
+  string creator = 1;
+
+  // metadata is any arbitrary string with a maximum length of 256 characters
+  // that includes or references metadata to attach to the project.
+  string metadata = 2;
+
+  // jurisdiction is the jurisdiction of the project. A jurisdiction has with
+  // the format: <country-code>[-<sub-national-code>[ <postal-code>]]
+  // The country-code must be 2 alphabetic characters, the sub-national-code
+  // can be 1-3 alphanumeric characters, and the postal-code can be up to 64
+  // alphanumeric characters. Only the country-code is required, while the
+  // sub-national-code and postal-code are optional and can be added for
+  // increased precision.
+  string jurisdiction = 3;
+
+  // reference_id is any arbitrary string used to reference the project with a
+  // maximum length of 32 characters.
+  string reference_id = 4;
+}
+
+// MsgCreateUnregisteredProjectResponse is the Msg/CreateUnregisteredProject response type.
+message MsgCreateUnregisteredProjectResponse {
+  // project_id is the unique identifier of the project.
+  string project_id = 1;
+}
+
+// MsgSubmitClassApplication is the Msg/SubmitClassApplication request type.
+message MsgSubmitClassApplication {
```

Comment:
Can we add a field representing the project term? In the context of a credit class, I believe a project will always have a defined project term (start date & end date), and it is then expected that the registry agent will issue credit batches for each vintage year during that project term.

Currently I don't think projects encode this data on-chain, but it might be useful to do if we want to restrict projects from enrolling from two credit classes of the same credit type during the same time period.